cmd/get.go

```diff
@@ -127,39 +112,56 @@ and provides a fast and easy alternative to a package manager.`,
 			os.Exit(2)
 		}()
 
-		outFilePath, finalName, err := get.Download(tool,
-			arch,
-			operatingSystem,
-			version,
-			dlMode,
-			progress)
-		if err != nil {
-			return errors.Wrap(err, "check with the vendor whether this tool is available for your system")
+		var outFilePath string
+		var localToolsStore []toolLocal
+
+		for _, tool := range userTools {
+			fmt.Printf("Downloading: %s\n", tool.Name)
+			outFilePath, _, err = get.Download(&tool,
+				arch,
+				operatingSystem,
+				version,
+				dlMode,
+				progress)
+			if err != nil {
+				return err
+			}
+
+			localToolsStore = append(localToolsStore, toolLocal{Name: tool.Name, Path: outFilePath})
+			fmt.Printf("Tool written to: %s\n\n", outFilePath)
 		}
 
-		fmt.Printf("Tool written to: %s\n\n", outFilePath)
+		t := template.New("Installation Instructions")
 
 		if dlMode == get.DownloadTempDir {
-			fmt.Printf(`Run the following to copy to install the tool:
+			t.Parse(`Run the following to copy to install the tool:
```

Comment:
I prefer string concatenation for maintenance, however if you want to add templates here, create a method to generate this text, and provide unit tests.

Including the empty case, 1 and many (more than 1).